spdx/package.py

```diff
@@ -271,11 +271,17 @@ def validate_str_fields(self, fields, optional, messages):
         return messages
 
     def validate_checksum(self, messages):
-        if self.check_sum is not None:
-            if not isinstance(self.check_sum, checksum.Algorithm):
-                messages.append(
-                    "Package checksum must be instance of spdx.checksum.Algorithm"
-                )
+        try:
+            if self.check_sum is not None:
+                if not isinstance(self.check_sum, checksum.Algorithm):
+                    messages.append(
+                        "Package checksum must be instance of spdx.checksum.Algorithm"
+                    )
+                # this check was added by the WhiteSouse PS Team ( for cases where SHA1 is not available at WhiteSource )
```

Comment:
Hi @nicoweidner - [will do](https://github.com/whitesource-ps/tools-python/issues/4)

Reply:
Hi @rammatzkvosky ! Since we didn't hear back for a while, @meretp took over this PR and put up a new version [here](https://github.com/spdx/tools-python/pull/270). Hope that is alright!